csm/core/services/hotfix_update.py

```diff
@@ -47,22 +47,24 @@ async def upload_package(self, file_ref, file_name):
 
         model = await self._get_renewed_model(const.SOFTWARE_UPDATE_ID)
         if model and model.is_in_progress():
+            Log.error("You cannot upload a new package while there is an ongoing update")
             raise InvalidRequest("You can't upload a new package while there is an ongoing update")
 
+        try:
+            file_ref.save_file(self.storage_path,file_name, True)
+        except Exception as e:
+            Log.error(f'Failed to save the package: {e}')
+            raise CsmInternalError(f'Failed to save the package: {e}')
+
+        Log.debug("Saving model for software update")
```

Comment:
Debug logs should contain some variable information and not just the static comment.

Reply:
Acknowledged.

Reply:
Comment resolved